Cargo.toml

```diff
@@ -5,6 +5,8 @@ authors = ["The Gtk-rs Project Developers"]
 
 [dependencies]
 libc = "0.1"
+gl = { version = "^0", optional = true }
+glutin = { version = "^0", optional = true }
```

Comment:
Would it be a good idea to clamp down on the allowed versions a bit? What if `glutin 0.5` removes `WindowBuilder`?